lib/fathead/gradle_docs/parse.js

```diff
@@ -0,0 +1,172 @@
+                    
+                });
+        }
+
+        console.log("** Gradle Fathead Processing Started at: " + new Date());
+        let $ = cheerio.load(fs.readFileSync(DOC_BASE_FILE));
+        let allUrls = [];
+        $('.overviewSummary td.colFirst > a').each(function(i, elem) {
+            allUrls[i] = $(this).attr("href");
+        });
+        console.log("Total urls to process: " + allUrls.length);
+        urlFetcher(allUrls);
+    }
+
+    // function to parse all php functions
+    function parserFunction(url, htmlData) {
+        let $ = cheerio.load(htmlData);
+        $('li.blockList > table').each(function(i, elem) {
+            //console.log($(this).html());
+            let children = $(this).children();
+            // extract category
+            let category = children.eq(0).children().eq(0).text().replace(/Summary|Types/g, '').trim();
+            // Process each row of the Category type
+            children.eq(2).children().each(function(i, elem) {
+                
+                // Process Type name and cleanup <T>'sand remove "."
+                let nameRowElement = $(this).children().eq(0);
+                let nameRowElementText = nameRowElement.text().replace(/<.*>/g, '');
+                nameRowElementText = nameRowElementText.split(".").join(" ");
+                // extract detail url for the current type
+                let nameRowElementShowMoreUrl = nameRowElement.find('a').attr('href');
+                nameRowElementShowMoreUrl = BASE_URL + nameRowElementShowMoreUrl.replace(/^(?:\.\.\/)+/, "")
+                
+                // Removing dup with same class and interface name
+                if (dupMap[nameRowElementText]) {
+                    nameRowElementText+= url.split("/").slice(0, -1).join(' ');
+                }else{
+                    dupMap[nameRowElementText] = category;
+                }
+                // Process details of the category
+                let detailRowElement = $(this).children().eq(1).children().eq(0);
+
+                // Code element wrapper
+                //detailRowElement.find('code').wrap(PRE_DOM_WRAPPER);
+
+                // Update anchor tag with absolute urls
+                detailRowElement.find('a').each(function(i, elem) {
+                    
+                    // let hrefVal = $(this).attr('href');
+                    // if(hrefVal.indexOf('http://') < 0 && hrefVal.indexOf('https://') < 0){
+                    //     hrefVal = hrefVal.replace(/^(?:\.\.\/)+/, "")
+                    //     $(this).attr('href', BASE_URL+hrefVal);
+                    // }
+
+                    // Removing <a> tags for now.
+                    $(this).replaceWith($(this).html());
+                });
+
+                //Wrap text nodes with <p>
+                //detailRowElement.contents().filter(parse_util.isTextNode).wrap(P_DOM_WRAPPER).end();
+
+                let detailNode = "<section class=\"prog__container\">" 
```

Comment:
nit: why not use single quotes so we don't need to escape the inner double quotes?